setup.py

```diff
@@ -30,7 +30,8 @@
         "Programming Language :: Python :: 3.9",
         "Topic :: Multimedia :: Graphics :: Capture :: Digital Camera",
     ],
-    packages=['picamera2', 'picamera2.encoders', 'picamera2.outputs', 'picamera2.previews', 'picamera2.allocators'],
+    packages=['picamera2', 'picamera2.devices.imx708', 'picamera2.encoders', 'picamera2.outputs', 'picamera2.previews',
```

Comment:
Do we need this? I thought just `picamera2.devices` would be enough provided you have set up the `devices/__init__.py` (which I think you have).